unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/GlobalUsersPositionMarker/Handlers/MarkersHandler.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+using Random = UnityEngine.Random;
+
+internal class MarkersHandler : IDisposable
```

Comment:
Could we add a documentation header here to explain what is the responsibility of this class? Thanks!